README.md

```diff
@@ -101,8 +101,21 @@ the system is intentionally less aggressive about automatically applying those
 (as can be seen in the example above).
 
 Sentence entries also contain a list of `tokens`. The tokens
-originate in the [Tokenizer package](https://github.com/mideind/Tokenizer)
-and their format is documented there.
+originate in the [Tokenizer package](https://github.com/mideind/Tokenizer) and contain the following fields:
+
+`i`: Character index of token start.
+`k`: Token type number, (WORD, DATEREL, AMOUNT, etc.). Mapping from numbers to strings is found in the documentation for the [Tokenizer package](https://github.com/mideind/Tokenizer).
+`o`: Original token text.
+`x`: Corrected text of token.
+
+Other possible fields:
+`s`: Stem of word. It can contain '-' if the stem is not found in BÍN and compound analysis has been used.
```

Comment:
More accurately, this is the "lemma", not the "stem". Suggest:

`'s': Lemma of word. It can contain '-' if the lemma does not appear in BÍN and the word has been identified as a compound word.`

Reply:
Much better, done